src/applications/public-outreach-materials/sass/public-outreach-materials.scss

```diff
@@ -0,0 +1,111 @@
+$formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
+
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-nav-sidebar";
+
+.outreach-asset-form-filters {
+  padding: 15px;
+  background: $color-gray-lightest;
+  max-width: 95%;
+  margin-bottom: 25px;
+  @media (max-width: $medium-screen) {
+    .vads-l-col--6,
+    .vads-l-col--5,
+    .vads-l-col--4 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  .vads-l-grid-container {
+    padding-left: 0;
+    padding-right: 0;
+  }
+  label {
+    margin-top: 0;
+  }
+  select {
+    max-width: 100%;
+  }
+}
+
+.asset-component-library {
+  @media (max-width: $medium-screen) {
+    .vads-l-col--6 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  .asset-card {
+    min-height: 450px;
+    float: left;
+    display: none;
+    &.show-type {
+      display: block;
+      &.hide-topic {
+        display: none;
+      }
+      &.pager-hide {
+        display: none;
+      }
+    }
+    &.show-topic {
+      display: block;
+      &.hide-type {
+        display: none;
+      }
+      &.pager-hide {
+        display: none;
+      }
+    }
+    &.hide-topic {
+      display: none;
+    }
+    &.hide-type {
+      display: none;
+    }
+    &.pager-hide {
+      display: none;
+    }
+
+    h3 {
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+    .asset-head-wrap {
+      max-width: 100%;
+      height: 212px;
+      overflow: hidden;
+      background: $color-primary-alt-darkest;
+      background-size: contain;
+      &.document-asset-wrap {
+        background: url(/img/megamenu/disability-illustration.png) center center;
+      }
+      &.video-asset-wrap {
+        background: url(/img/megamenu/health-care-illustration.png) center center;
+      }
+      img {
+        margin: 0 auto;
+        display: block;
+      }
+    }
+    .asset-body-wrap {
+      background: $color-gray-lightest;
+      height: 250px;
+      padding: 15px;
+      .asset-body-text,
+      .asset-category-text,
+      .asset-filetype-text {
+        padding-bottom: 12px;
+      }
+      .capitalize-first {
+        text-transform: capitalize;
+      }
+    }
+  }
+}
+
+@media (max-width: $medium-screen) {
+  .max-control {
```

Comment:
Could set `.max-control` to be display none by default and then add `medium-screen:vads-u-display--block` to the element, I think. Might be `small-desktop-screen:vads-u-display--block`, not sure exactly what the page looks like.

In general, we try to write styles mobile-first, meaning that styles are written for small screen sizes first, then media queries are written to adjust them for larger screen sizes. This is how the design system we're built on is meant to be used, so it reduces friction when we write that way, too. In practice that means avoiding `max-width` queries in favor of `min-width`.

Reply:
:+1: @jbalboni